src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultClaimUidExtractor.cs

```diff
@@ -38,6 +38,17 @@ public string ExtractClaimUid(ClaimsIdentity claimsIdentity)
         // Internal for testing
         internal static IEnumerable<string> GetUniqueIdentifierParameters(ClaimsIdentity claimsIdentity)
```

Comment:
@blowdart the code currently never goes beyond the first `ClaimsIdentity`. Caller uses `User?.Identity as ClaimsIdentity` and not a search through `User?.Identities`. Is that significant?